flipt-client-js/rollup.config.js

```diff
@@ -0,0 +1,80 @@
+import typescript from '@rollup/plugin-typescript';
+import { wasm } from '@rollup/plugin-wasm';
+import { copyFileSync, mkdirSync, unlinkSync } from 'fs';
+import { dirname } from 'path';
+import glob from 'glob';
+
+// Custom plugin to copy WASM files to output directories
+const copyWasmPlugin = (targetDir) => ({
```

Comment:
this is required to copy all the wasm files from the root `dist` to `dist/browser` and `dist/node` so that the code generated by wasm bindgen can correctly find the files. i don't understand why this is necessary since we also inline the wasm using rollup.. but alas.. it doesn't seem to work without doing this if we want to keep everything in separate dirs

without this we get this error:

```
Module not found: Can't resolve 'flipt_engine_wasm_js_bg.wasm'
  641 |
  642 |     if (typeof module_or_path === 'undefined') {
> 643 |         module_or_path = new URL('flipt_engine_wasm_js_bg.wasm', import.meta.url);
      |                          ^
  644 |     }
  645 |     const imports = __wbg_get_imports();
  646 |
```